extension/api.js

```diff
@@ -83,10 +88,20 @@
 			url += 'api/v3/notifications';
 		}
 
-		url += participating;
+		url += `?${query.join('&')}`;
 
 		xhr('GET', url, opts, (data, status, response) => {
```

Comment:
Here is the quote:
> I can't think of a smarter approach. If you need the total count of notifications -- you can get that with one request by setting the per_page to 1 and looking at the link header. But if you want to fetch all the notifications -- you'll need to go through all the pages. You can set per_page to 100 in that case to get as many notifications as possible in a single page. You could perhaps be smart about this and use the since parameter to fetch only notifications that were updated since the last time you fetched the notifications. In most cases, there will be less than 100 of those, so you'd be making only 1 request.